src/package-manifest.ts

```diff
@@ -29,6 +29,8 @@ export type ValidatedPackageManifest = {
   readonly [PackageManifestFieldNames.Version]: SemVer;
   readonly [PackageManifestFieldNames.Private]: boolean;
   readonly [PackageManifestFieldNames.Workspaces]: string[];
+  readonly [PackageManifestFieldNames.Dependencies]: Record<string, string>;
+  readonly [PackageManifestFieldNames.PeerDependencies]: Record<string, string>;
```

Comment:
Ah, this would make sense — but it seems like these can be found under different names:

``` suggestion
  readonly [PackageManifestDependenciesFieldNames.Production]: Record<string, string>;
  readonly [PackageManifestDependenciesFieldNames.Peer]: Record<string, string>;